README.md

```diff
@@ -1,59 +1,53 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# AWESOME BOOKS
 
-> One paragraph statement about the project.
+> Awesome Books allows to add and delete book names to a local storage using javascript objects.
 
-![screenshot](./app_screenshot.png)
+# Notable Features
+> This project is refactored using ES6 and modules for easier long term maintenance.
 
 Additional description about the project and its features.
 
+# Screenshots
+![DESKTOP VIEW](./images/awesome-books.png)
+![MOBILE VIEW](./images/awesome-books-mobile.png)
+![DROP DOWN MENU MOBILE VIEW](./images/awesome-books-drop-down.png)
```

Comment:
- [ ] Please the links to the screenshots are broken, kindly fix that as this will make your `readme file` look more professional.